src/hooks/useRenderCampaigns.tsx

```diff
@@ -16,8 +16,10 @@ function RecommendationComponentWrapper(props: {
   })
 }
 
-function injectCampaigns(data: ActionResponse, api: NostoClient) {
-  api.placements.injectCampaigns(data.recommendations)
+function injectCampaigns(data: ActionResponse) {
+  window.nostojs(api => {
```

Comment:
Maybe include it under a check whether window.nostojs is present and log a warning that nostojs is not initialized?